doc/generated/libpmemfile-posix.3

```diff
@@ -36,13 +36,10 @@
 \f[B]libpmemfile\-posix\f[] \- user space persistent memory aware file
 system API
 .SH SYNOPSIS
-.IP
-.nf
-\f[C]
-#include\ <libpmemfile\-posix.h>
-cc\ \-std=gnu99\ ..\ \-lpmemfile\-posix\ \-lpmemobj
-\f[]
-.fi
+.PP
+\f[C]#include\ <libpmemfile\-posix.h>\f[]
+.PP
+\f[C]cc\ \-std=gnu99\ ..\ `pkg\-config\ libpmemfile\-posix\ \-\-cflags\ \-\-libs`\f[]
```

Comment:
If we'll ever start using c99 features in the header file (we already export some non-posix functions, so it's possible) then your proposed change would have to be reverted and this can affect applications. If we say we require c99 from the very beginning then no one should be surprised when we actually start using it. c99 is old enough.